content/PGP/Importing_keys.adoc

```diff
@@ -289,5 +289,5 @@ And as a last step we've now moved the Authentication key to the YubiKey.
   gpg> quit
   Save changes? (y/N) y
 
-After this the keyring is saved, now no longer containing the real secret key,
+After this the keyring is saved, and no longer contains the real secret key,
```

Comment:
See, you are pretty good at this documentation thing 😉 

Done with a tiny nit (And at that...)